pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -417,7 +429,7 @@ func (s *DiscoveryServer) handleUpdates(stopCh <-chan struct{}) {
 					pushCounter, debouncedEvents,
 					quietTime, eventDelay, fullPush)
 
-				go s.doPush(fullPush)
+				fn(fullPush)
```

Comment:
does this result in a goroutine?

Reply:
This should function exactly the same as it did previously. Before we called a goroutine, now we call a function which just calls the goroutine.